README.md

```diff
@@ -143,7 +143,7 @@ Note that "msauthv3" scheme is needed when compiling your app with Xcode 11 and
 	<string>msauthv3</string>
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->